Week 04/Lecture 08/Assignment 03/src/main/java/com/lecture8/EmployeeCRUD/config/DataSourceConfig.java

```diff
@@ -0,0 +1,43 @@
+package com.lecture8.EmployeeCRUD.config;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import javax.sql.DataSource;
+
+@Configuration
+public class DataSourceConfig {
+
+    @Bean(name = "dataSource1")
+    public DataSource dataSource1() {
+        DataSourceBuilder<?> dataSourceBuilder = DataSourceBuilder.create();
+        dataSourceBuilder.driverClassName("org.postgresql.Driver");
+        dataSourceBuilder.url(System.getenv("DB_URL"));

```

Comment:
It's good to put url, username, pass in prop file, but it's better to use @Value to get value from prop file